keyboards/charue/sunsetter_r2/config.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 Charue Design
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+// 0x4344 = CD for Charue Design
+#define VENDOR_ID    0x4344
+// 0x5335 = S2 for Sunsetter R2
+#define PRODUCT_ID   0x5335
+// 0x0001 = 1 for Base version
+#define DEVICE_VER   0x0001
+#define MANUFACTURER Charue Design
+#define PRODUCT      Sunsetter R2
+
+/* Key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 17
+
+/* Keyboard Matrix Assignment s*/
+#define MATRIX_ROW_PINS { B3, B2, F4, F5, F6 }
+#define MATRIX_COL_PINS { F0, F1, F7, B1, D0, D1, D2, D3, D5, D4, D6, D7, B4, B5, B6, C6, C7 }
+#define UNUSED_PINS { B7 }
+
+/* COL2ROW, ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+#define LED_CAPS_LOCK_PIN B0
+
+/* RGB */
+#define RGB_DI_PIN E6
+#ifdef RGB_DI_PIN
+#    define RGBLED_NUM 10
+#    define RGBLIGHT_ANIMATIONS
```

Comment:
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

Reply:
Of course, but I thought that `RGBLIGHT_ANIMATIONS` enables all of them. Is there an issue with this?

Reply:
It does, and that is part of the problem.  Especially on AVR.  

So, explicitly enabled helps reduce the amount of issues.